modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -60,6 +61,13 @@ public interface MetaStorageManager extends IgniteComponent {
      */
     CompletableFuture<Entry> get(ByteArray key, long revUpperBound);
 
+    /**
+     * Retrieves entries for the given key and bound revisions.
```

Comment:
Please add detailed javadoc. 
-  boundaries included/excluded?
- Result is a list of values ordered by revisions, isn't it?
etc.

Reply:
I've updated javadoc. I tried to describe it more clearly. Not sure if I succeeded. But I can't write better.